appinventor/components/src/com/google/appinventor/components/runtime/Canvas.java

```diff
@@ -433,27 +433,19 @@ public CanvasView(Context context) {
      * Create a bitmap showing the background (image or color) and drawing
      * (points, lines, circles, text) layer of the view but not any sprites.
      */
-    private Bitmap buildCache() {
-      // First, try building drawing cache.
-      setDrawingCacheEnabled(true);
-      destroyDrawingCache();      // clear any earlier versions we have requested
-      Bitmap cache = getDrawingCache();  // may return null if size is too large
-
-      // If drawing cache can't be built, build a cache manually.
-      if (cache == null) {
+    private Bitmap createBitmap() {
         int width = getWidth();
         int height = getHeight();
-        cache = Bitmap.createBitmap(width, height, Bitmap.Config.ARGB_8888);
-        android.graphics.Canvas c = new android.graphics.Canvas(cache);
+        Bitmap currentBitmap = Bitmap.createBitmap(width, height, Bitmap.Config.ARGB_8888);
+        android.graphics.Canvas c = new android.graphics.Canvas(currentBitmap);
         layout(0, 0, width, height);
         draw(c);
-      }
-      return cache;
+        return currentBitmap;
```

Comment:
Please fix the indentation here. Our code style is to indent a new scope by 2 spaces only.